features/unsupported/USBHost/USBHostMSD/USBHostMSD.cpp

```diff
@@ -29,13 +28,16 @@
 #define GET_MAX_LUN             (0xFE)
 #define BO_MASS_STORAGE_RESET   (0xFF)
 
-USBHostMSD::USBHostMSD(const char * rootdir) : FATFileSystem(rootdir)
+USBHostMSD::USBHostMSD()
```

Comment:
The BlockDevice was not existing when the initial version has been developped. 
When BlockDevice and  fatfilesystem evolution has been introduced, USBHostMSD was no more working (USB: MSD Device #4131) . 
Since USB stick is exchanging block write , block read on USB.
To let application using  fatfilesystem on this BlockDevice seems in line with the mbed filesystem evolution.